network.c

```diff
@@ -1356,8 +1356,8 @@ struct iio_context * network_create_context(const char *host)
 			errno = -ret;
 			return NULL;
 		}
-
 		iio_snprintf(port_str, sizeof(port_str), "%hu", port);
+		fprintf(stderr, "Looking for %s:%s\n", addr_str,port_str);
```

Comment:
this looks like a debug print();
maybe should be wrapped in a `DEBUG()` or similar macro?

Reply:
yeah, I left this once, so I would remember to completely remove it before submission.